ide/project.dependency/nbproject/project.xml

```diff
@@ -156,7 +164,9 @@
                     </test-dependency>
                 </test-type>
             </test-dependencies>
-            <public-packages/>
+            <public-packages>
```

Comment:
I still have some concerns (as raised https://lists.apache.org/thread/ztc5vhdoooggfvzklnbgcozn6w6r11c4) about making this API public at the moment.

Reply:
Thanks - this was not the intent, the dependencies should be published separately; will hide the package again - but still intend to publish the API, ideally this calendar year (I have one more refactoring to go).

The Project Reload is a separate, it's just hosted here. I hope it will move soon to public API into base project.api module.